packages/@react-types/combobox/src/index.d.ts

```diff
@@ -34,16 +35,29 @@ export interface ComboBoxProps<T> extends CollectionBase<T>, SingleSelection, In
   //  * @default 'suggest'
   //  */
   // completionMode?: 'suggest' | 'complete',
-  /**
-   * The interaction required to display the ComboBox menu.
-   * @default 'input'
-   */
-  menuTrigger?: MenuTriggerAction,
+ /**
+  * The interaction required to display the ComboBox menu.
+  * @default 'input'
+  */
+  menuTrigger?: MenuTriggerAction
+}
```

Comment:
menuTrigger should also follow and move to AriaComboBoxProps right?

Reply:
Per our chat, leaving menuTrigger here because it is used by stately.